[Term Entry] JS Window Function: resizeTo() (#6167)
Codecademy/docs: content/javascript/concepts/window/terms/resizeTo/resizeTo.md

---

> Title: 'resizeTo()'
> Description: 'It is used to resize the window dynamically'
> Subjects: 
>     - 'Web Development'
>     - 'Computer Science'
> Tags:
>     - 'Arguments'
>     - 'Functions'
>     - 'Parameters'
> CatalogContent:
>     - 'Introduction-to-Javascipt'
>     - 'paths/front-end-engineer-career-path'
> 

Reviewer: ```suggestion
Title: '.resizeTo()'
Description: 'Resizes the browser window to the specified width and height in pixels.'
Subjects: 
    - 'Web Development'
    - 'Computer Science'
Tags:
    - 'Arguments'
    - 'Functions'
    - 'Parameters'
CatalogContent:
    - 'Introduction-to-javascipt'
    - 'paths/front-end-engineer-career-path'
```

Reviewer: Description should start with a verb

---

> The ***resizeTo()*** function is used to change the size of a window to a new width and height.

Reviewer: ```suggestion
The **`.resizeTo()`** function resizes the browser window to the specified width and height in pixels.
```

Reviewer: term should be addded in backticks (``)

Author: The backticks(``) has been used now.

## Syntax

> ```js

Reviewer: ```suggestion

```pseudo
```

Reviewer: syntax is added in pseudo blocks

Author: Syntax is fixed

>         window.resizeTo(width, height);

Reviewer: ```suggestion
window.resizeTo(width, height);
```

Author: Syntax fixed

```

> - `width` : It will be the new total number of pixels of the window width after the ***resizeTo*** function has been executed.
> 
> - `height` : It will be the new total number of pixels of the window height after the  ***resizeTo*** function has been executed.

Reviewer: ```suggestion
- `width` : The new width of the window in pixels after resizing.
- `height` : The new height of the window in pixels after resizing.
```

## Example

> 
> 

Reviewer: ```suggestion

```

Reviewer: Leave a single line only

Author: A single line has been left
